Common/src/main/java/subaraki/pga/capability/ScreenData.java

```diff
@@ -58,8 +58,6 @@ public ResourceLocation lookupResloc() {
         if (viewingScreen != null) {
             if (cachedResLoc == null) {
                 cachedResLoc = new ResourceLocation(viewingScreen.getResLoc());
-            } else if (!cachedResLoc.toString().equals(viewingScreen.getResLoc())) {
-                cachedResLoc = new ResourceLocation(viewingScreen.getResLoc());
             }
```

Comment:
can these conditions be merged ? so that the cached resourcelocation only gets set when it is either null, or not the same as the already existing one, to prevent creating new objects when not needed ?

Reply:
You're right I merged them incorrectly. Should be fixed in the next commit.